File: blinkeye-framewk-comun-dominio/src/main/java/net/royal/spring/sistema/dominio/dto/DtoComunEmailTest.java
package net.royal.spring.sistema.dominio.dto;

import net.royal.spring.framework.modelo.correo.EmailConfiguracion;
import net.royal.spring.framework.modelo.correo.EmailTransaccion;

public class DtoComunEmailTest {
	public EmailConfiguracion config;
	public EmailTransaccion correo;
	
	public DtoComunEmailTest() {
		config=new EmailConfiguracion();
		correo=new EmailTransaccion();
	}
	
	public EmailConfiguracion getConfig() {
		return config;
	}
	public void setConfig(EmailConfiguracion config) {
		this.config = config;
	}
	public EmailTransaccion getCorreo() {
		return correo;
	}
	public void setCorreo(EmailTransaccion correo) {
		this.correo = correo;
	}	
}

File: web-deteccion/src/app/royal/modulos/framework-comun/sy/dominio/dto/DtoComunEmailTest.ts
import { EmailConfiguracion } from 'src/app/royal/framework/modelo/correo/EmailConfiguracion';
import { EmailTransaccion } from 'src/app/royal/framework/modelo/correo/EmailTransaccion';

export class DtoComunEmailTest{	
    constructor() {
        this.config = new EmailConfiguracion();
        this.correo = new EmailTransaccion();
    }
    config: EmailConfiguracion;
    correo: EmailTransaccion;
}